pkg/pureutils/pure_types.go

```diff
@@ -0,0 +1,96 @@
+package pureutils
```

Comment:
Why did you move this file from test path to pureutils. Are we going to use it anywhere outside of tests?

Reply:
It was a code review comment from above. I think the goal is to keep more "utility function" code in `pkg` while keeping the `tests` folder for actual test code/logic?